packages/components/src/components/hero/hero-logo.tsx

```diff
@@ -13,20 +18,22 @@ export function HeroLogo({ children, className, ...rest }: HeroLogoProps) {
   return (
     <div className={cn('relative', className)} {...rest}>
       {cloneElement(children, {
-        fill: `url(#${GRADIENT_WHITE_ID})`,
+        stroke: `url(#${GRADIENT_WHITE_INVERTED})`,
+        fill: `url(#${GRADIENT_BLUE})`,
```

Comment:
This seems to have been oversimplified previously.
I kept thinking there's something off with the Gateway landing and it ended up being the color of the hero emblem.